packages/botonic-cli/src/commands/train.ts

```diff
@@ -1,44 +1,103 @@
 import { Command, flags } from '@oclif/command'
 import colors from 'colors'
-import * as path from 'path'
+import { existsSync } from 'fs'
+import { join } from 'path'
 
 import { spawnNpmScript, track } from '../utils'
 
+enum TASKS {
+  NER = 'ner',
+  TEXT_CLASSIFICATION = 'text-classification',
+}
+
 export default class Run extends Command {
-  static description = 'Train your bot with NLU'
+  static description = 'Train your bot with NLP'
 
   static examples = [
-    `$ botonic train
-    TRAINING MODEL FOR {LANGUAGE}...
+    `$ botonic train [--task=<${Object.values(TASKS).join('|')}>]
+    TRAINING MODEL...
     `,
   ]
 
   static flags = {
-    lang: flags.string(),
+    task: flags.string(),
   }
 
   static args = []
 
   async run(): Promise<void> {
-    const botonicNLUPath: string = path.join(
-      process.cwd(),
-      'node_modules',
-      '@botonic',
-      'nlu'
-    )
-    try {
-      await import(botonicNLUPath)
-    } catch (e) {
-      console.log(e)
-      console.log(
-        colors.red(
-          `You don't have @botonic/plugin-nlu installed.\nPlease, install it with the following command:`
-        )
+    const { flags } = this.parse(Run)
+    const task = flags.task
+    if (task) {
```

Comment:
minimize if/checks and remember SRP.
create Task classes Ner and classification and you'll see that everything is simpler.

```
const ALL_TASKS = {
NER : new NerTask()
classification: new ClassificationTask()
}

getTaskByName(taskName): Task {
 if (!ALL_TASKS.taskName) {
    move here the contents of logInvalidTask 
}
async run() {

  const tasks = flags.task ?  [getTaskByName(flags.task)] : ALL_TASKS.values()
 tasks.forEach(t-> t.run()) 
}
```

Reply:
Not sure if I did something similar to what you expected, I would thank you very much if you can take a look at it. :pray: